Marketplace/emoji-tablet/behaviors/Edible.js

```diff
@@ -29,7 +29,8 @@
     var checkIfNearHead = function () {
         var position = Entities.getEntityProperties(_this.entityID, 'position').position;
         var avatarHeadPosition = MyAvatar.getJointPosition("Head");
-        if (isWithinDistance(position.y, avatarHeadPosition.y) &
+        if (isWithinDistance(position.x, avatarHeadPosition.x) &
+            isWithinDistance(position.y, avatarHeadPosition.y) &
```

Comment:
You could use `Vec3.distance(position, MyAvatar.getJointPosition("Head")) < DISTANCE_WITHIN` to cut down on lines